src/switch_ivr_async.c

```diff
@@ -807,7 +808,7 @@ static switch_bool_t write_displace_callback(switch_media_bug_t *bug, void *user
 			switch_core_file_close(&dh->fh);
 
 			if (session && (channel = switch_core_session_get_channel(session))) {
-				switch_channel_set_private(channel, dh->file, NULL);
+				switch_channel_set_private(channel, dh->id, NULL);
```

Comment:
By default, file and id are the same and it uses to set the private variable to store the context handle for the media bug. Since we now introduce to support setting the id instead of the filename, they can pass this id to stop the displace. This works well with base64 payload, previously this how they start/stop displace
```
uuid_displace <uuid> start {extension=mp3}base64://<very large data>
...
uuid_displace <uuid> stop {extension=mp3}base64://<very large data>
```

With the new id implementation, they can do this:

```
uuid_displace <uuid> start {extension=mp3,id=<unique-id>}base64://<very large data>
...
uuid_displace <uuid> stop <unique-id>
```

Reply:
Got it. Thanks!